src/ctap/test_helpers/env.rs

```diff
@@ -0,0 +1,49 @@
+use crate::clock::CtapInstant;
+use crate::ctap::command::{
+    AuthenticatorAttestationMaterial, AuthenticatorConfigParameters,
+    AuthenticatorVendorConfigureParameters, Command,
+};
+use crate::ctap::data_formats::ConfigSubCommand;
+use crate::ctap::status_code::Ctap2StatusCode;
+use crate::ctap::{key_material, Channel, CtapState};
+use crate::env::Env;
+
+// In tests where we define a dummy user-presence check that immediately returns, the channel
+// ID is irrelevant, so we pass this (dummy but valid) value.
+const DUMMY_CHANNEL: Channel = Channel::MainHid([0x12, 0x34, 0x56, 0x78]);
+
+pub struct SetupEnterpriseAttestationResponse {
+    pub attestation_material: AuthenticatorAttestationMaterial,
+}
+
+pub fn setup_enterprise_attestation(
+    state: &mut CtapState,
+    env: &mut impl Env,
+) -> Result<SetupEnterpriseAttestationResponse, Ctap2StatusCode> {
+    let config_params = AuthenticatorConfigParameters {
+        sub_command: ConfigSubCommand::EnableEnterpriseAttestation,
+        sub_command_params: None,
+        pin_uv_auth_param: None,
+        pin_uv_auth_protocol: None,
+    };
+
+    let dummy_key = [0x41u8; key_material::ATTESTATION_PRIVATE_KEY_LENGTH];
```

Comment:
Do we really need the `u8` suffix? It's used right below so I would assume inference is able to do this.

Reply:
fixed